### PR TITLE
univa: code cleanup

### PR DESCRIPTION
**Pull request by bernt-matthias**
some debug messages somehow survived

### CODE REVIEW

File: lib/galaxy/jobs/runners/univa.py [hunk 1]
```diff
@@ -86,21 +79,21 @@ def _complete_terminal_job(self, ajs, drmaa_state, **kwargs):
             mem_wasted = extinfo["memory_wasted"]
             slots = extinfo["slots"]
 
-            # log.debug("UnivaJobRunner:_complete_terminal_job ({jobid}) memviolation {mv}".format(jobid=ajs.job_id, mv=memviolation))
+                # log.debug("UnivaJobRunner:_complete_terminal_job ({jobid}) memviolation {mv}".format(jobid=ajs.job_id, mv=memviolation))
```
Comment (mvdbeek) [inline lib/galaxy/jobs/runners/univa.py]: Can you just drop the comment ?
Else you need to drop the extra indentation here.

File: lib/galaxy/jobs/runners/univa.py [hunk 2]
```diff
@@ -1,6 +1,7 @@
 """
-Kind of fail save job control via the DRMAA API.
-Tested for UNIVA grid engine
+Kind of fail save job control via the DRMAA API / qstat and qacct.
```
Comment (mvdbeek) [inline lib/galaxy/jobs/runners/univa.py]: fail-safe ?

**Review by mvdbeek — COMMENTED** [review]
You also need to drop the galaxy.model.Job import (https://travis-ci.org/galaxyproject/galaxy/jobs/478768574)

File: lib/galaxy/jobs/runners/univa.py (hunk 1 above)
Comment (bernt-matthias) [reply lib/galaxy/jobs/runners/univa.py]: Will drop the extra indentation. If debugging is necessary its easier to renable. I guess that some debugging might be necessary if others start using it. 

Will do it everywhere.

File: lib/galaxy/jobs/runners/univa.py (hunk 2 above)
Comment (bernt-matthias) [reply lib/galaxy/jobs/runners/univa.py]: Don't know how this got there :) .. consider it gone.

**Review by mvdbeek — APPROVED** [review]

**Comment by mvdbeek** [comment]
Thanks @bernt-matthias !

**Pull request merged**